x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/import_rules_route.ts

```diff
@@ -47,7 +45,7 @@ import { PartialFilter } from '../../types';
 
 type PromiseFromStreams = ImportRulesSchemaDecoded | Error;
 
-const CHUNK_PARSED_OBJECT_SIZE = 10;
+const CHUNK_PARSED_OBJECT_SIZE = 300;
```

Comment:
👍

Reply:
I will have to drop this down to 50 otherwise with my tests importing those 203 rules, some of the rules fail to import because of timeouts on elasticsearch.